src/servers/http_server.cc

```diff
@@ -2471,6 +2497,11 @@ HTTPAPIServer::InferRequestClass::InferResponseComplete(
   HTTPAPIServer::InferRequestClass* infer_request =
       reinterpret_cast<HTTPAPIServer::InferRequestClass*>(userp);
 
+  // Always specify supported compression as Accept-Encoding
+  evhtp_headers_add_header(
```

Comment:
The response doesn't need the Accept-Encoding header

Reply:
How would the client knows what compression type is supported by the server?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation#server-driven_content_negotiation

It looks like we should return 415 if client has Content-Encoding that Triton can support. That will tell the client that they need to try the request again with a different Content-Encoding. We don't need to automatically do the retry in our client libraries.

Reply:
So only on client sending content with unsupported type, the server returns 415 and attach the supported type as `Accept-Encoding` in the response header?

Reply:
Client-side compression negitiation doesn't seem to be very well standardized. It should be ok to include Accept-Encoding in that error response.